app/posts/find-teacher-training/2025-11-05-improving-filtering-and-sorting.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Improving how candidates can filter and sort courses
+description: Researching filter components across government and designing and testing a solution to help candidates filter courses and see results.
+date: 2025-11-05
+tags:
+  - find
+  - filters
+  - filterable find
+---
+
+> **Relates to:**
+> [Filterable Find postgraduate training courses (Find) concept](/find-teacher-training/filterable-find-concept/) - October 2022
```

Comment:
You can place the related links in the front matter so they load into the sidebar.

```suggestion
related:
  items:
    - text: Filterable Find postgraduate training courses (Find) concept
      description: October 2022
      href: /find-teacher-training/filterable-find-concept/
---
```